tests/App/config/services.yml

```diff
@@ -38,3 +38,8 @@ services:
         arguments: [~, Sonata\AdminBundle\Tests\App\Model\Foo, ~]
         tags:
             - {name: sonata.admin, manager_type: test, label: Foo}
+
+    Sonata\AdminBundle\Tests\App\Admin\EmptyAdmin:
+        arguments: [~, Sonata\AdminBundle\Tests\App\Model\Foo, ~]
```

Comment:
```suggestion
        arguments: [sonata.admin.foo, Sonata\AdminBundle\Tests\App\Model\Foo, ~]
```

Reply:
What's for ?

Reply:
The default value you're referencing is set at `AddDependencyCallsCompilerPass::process()`, but IMO we should rely as less as possible on features which aren't related to the tested behavior here.
https://github.com/sonata-project/SonataAdminBundle/blob/87bd4e83078d5c6b0196509399f52dc22ec523c6/src/Admin/AbstractAdmin.php#L576-L580

Reply:
Tests are failing with the change you requested @phansys.

It seems better to me to rely on the default value.

Reply:
No problem then.
IMO, this test reflects that we're relying on some hidden invalid values, but I understand that's not directly related with what you're trying to prove.